benchmarks/luafun/bench.lua

```diff
@@ -0,0 +1,136 @@
+local fun = require 'fun'
+local jitlog_capnp = require "jitlog_capnp"
+local bor = bit.bor
+local lshift = bit.lshift
+-- Enable table.new function

```

Comment:
I'm not sure this comment adds anything. Maybe remove?

Reply:
I just added it because the require result is not stored into anything it might not be obvious why.

Reply:
I assumed this was a standard Lua idiom?

Reply:
I've only seen LuaJIT do it for one or two functions

Reply:
OK.